common-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/common/client/LoginPage.java

```diff
@@ -311,6 +313,12 @@ public void onChanged(ChangedEvent event) {
         this.layout.draw();
     }
 
+    // console log in GWT
+    // wrapper to console.log following this: https://stackoverflow.com/questions/9247424/how-to-print-to-the-console-in-gwt 
+    native void consoleLog(String message) /*-{
+                                           console.log(message);
+                                           }-*/;
+
```

Comment:
Do you need this?

Reply:
I added it to log (debug), as in GWT it is not possible to do a proper console.log, @amouhoub faced this same issue.
I will remove it.